asset/src/kafka_reader/schema.ts

```diff
@@ -73,6 +91,11 @@ export default class Schema extends ConvictSchema<KafkaReaderConfig> {
                 doc: 'Name of partition assignment strategy to use when elected group leader assigns partitions to group members.',
                 default: '',
                 format: ['range', 'roundrobin', '']
+            },
+            _encoding: {
```

Comment:
This is automatically added to to all operation schemas in `job-components` - we don't need to specify this twice.